pytorch/inference/docker/1.12/py3/cu113/Dockerfile.gpu

```diff
@@ -2,10 +2,10 @@ ARG PYTHON=python3
 ARG PYTHON_VERSION=3.8.13
```

Comment:
I dont see binary changes for inference cpu dockerfiles

Reply:
CPU inference binary change has been merged into master by the 1.12.1_ec2 PR